packages/react-composites/playwright.config.hermetic.ts

```diff
@@ -11,10 +11,7 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 3 : undefined,
 
   // All these tests are hermetic. It is safe to run them in parallel.
-  fullyParallel: true,
-
-  // TODO(prprabhu): Remove retries once super flakey tests are fixed or removed.
```

Comment:
Yay if we can do this! Might be worth moving to its own PR and running the stress test action over it

Reply:
Removed here because of the triplicate screen checks wasting time on the build.